taskRequests/script.js

```diff
@@ -63,7 +37,7 @@ async function getTaskRequests() {
 
     showMessage('ERROR', ErrorMessages.SERVER_ERROR);
   } catch (e) {
-    console.log(e);
+    console.error(e);
   } finally {
```

Comment:
I think we should handle error in catch block like - 
`showMessage('ERROR', ErrorMessages.SERVER_ERROR);`